tests/shakedown/shakedown/dcos/service.py

```diff
@@ -263,12 +266,12 @@ def mesos_task_not_present_predicate(task_name):
     return get_mesos_task(task_name) is None
 
 
-def wait_for_mesos_task(task_name, timeout_sec=120):
-    return time_wait(lambda: mesos_task_present_predicate(task_name), timeout_seconds=timeout_sec)
+def wait_for_mesos_task(task_name):
+    assert_that(lambda: mesos_task_present_predicate(task_name), eventually(equal_to(True)))
```

Comment:
So instead of waiting for 120s we wait for...3s? O_O
I also liked the fact that previously we could control the time in the call but I guess that's not absolutely necessary.

Reply:
Dang, you are right. Will do.